evolve_text.py

```diff
@@ -170,7 +197,7 @@ def evolve_string(text):
 
     # Get configured toolbox and create a population of random Messages
     toolbox = get_toolbox(text)
-    pop = toolbox.population(n=300)
+    pop = toolbox.population(n=10)
```

Comment:
I had to switch this back to 300 to actually get to the proper result, however I think you switched it to 10 for testing which is fine